extensions/github/src/links.ts

```diff
@@ -70,15 +70,15 @@ function getFileAndPosition(context: LinkContext): IFilePosition | INotebookPosi
 			const cellIndex = cell?.index ?? vscode.window.activeNotebookEditor.selection.start;
 
 			let range;
-			if (lineNumber !== undefined) {
+			if (lineNumber && (!vscode.window.activeTextEditor || vscode.window.activeTextEditor.selection.isEmpty || !vscode.window.activeTextEditor.selection.contains(new vscode.Position(lineNumber - 1, 0)))) {
```

Comment:
You mean `line !== undefined` so 0 isn't ignored?

Reply:
`lineNumber` starts at 1 not 0...but it's probably better to do an explicit check anyways 👍